src/test/java/pagefactory/HomePage.java

```diff
@@ -16,6 +17,9 @@ public class HomePage extends BasePage {
     @FindBy(css = "div.success.show")
     WebElement popUpNotification;
 
+    @FindBy(css = "li a.songs")
+    WebElement allSongsList;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```